api-spec/protobuf/ark/v1/service.proto

```diff
@@ -185,47 +135,45 @@ message SubmitTreeSignaturesResponse {}
 message SubmitSignedForfeitTxsRequest {
   // Forfeit txs signed by the user.
   repeated string signed_forfeit_txs = 1;
-  // The user has to sign also the round tx if he registerd a boarding UTXO.
-  optional string signed_round_tx = 2;
+  // The user has to sign also the commitment tx if he registerd a boarding UTXO.
+  optional string signed_commitment_tx = 2;
 }
 message SubmitSignedForfeitTxsResponse {}
 
 message GetEventStreamRequest {}
 message GetEventStreamResponse {
   oneof event {
     BatchStartedEvent batch_started = 1;
-    RoundFinalizationEvent round_finalization = 2;
-    RoundFinalizedEvent round_finalized = 3;
-    RoundFailed round_failed = 4;
-    RoundSigningEvent round_signing = 5;
-    RoundSigningNoncesGeneratedEvent round_signing_nonces_generated = 6;
-    BatchTreeEvent batch_tree = 7;
-    BatchTreeSignatureEvent batch_tree_signature = 8;
+    BatchFinalizationEvent batch_finalization = 2;
+    BatchFinalizedEvent batch_finalized = 3;
+    BatchFailed batch_failed = 4;
+    TreeSigningStartedEvent tree_signing_started = 5;
+    TreeNoncesAggregatedEvent tree_nonces_aggregated = 6;
+    TreeTxEvent tree_tx = 7;
+    TreeSignatureEvent tree_signature = 8;
   }
 }
 
-/* Out-of-Round Transaction API messages */
-
-message SubmitOffchainTxRequest {
-  string virtual_tx = 1;
+message SubmitTxRequest {
+  string signed_virtual_tx = 1;
   repeated string checkpoint_txs = 2;
 }
-message SubmitOffchainTxResponse {
-  string signed_virtual_tx = 1;
-  string txid = 2;
+message SubmitTxResponse {
```

Comment:
I still find confusing having various messages and fields referring to same thing: Tx, ArkTransaction, virtual_tx